src/Database/Inscriptions/Methods.ts

```diff
@@ -38,12 +38,12 @@ export const inscriptions = {
  | Core Methods
  |--------------------------------------------------------------------------------
  |
- | Flexible core mongodb methods for the collection. These methods provides a 
+ | Flexible core mongodb methods for the collection. These methods provides a
  | unified passthrough to the collection directly for querying.
  |
  */
 
-async function insertMany(inscriptions: Inscription[], chunkSize = getChunkSize(inscriptions.length)) {
+async function insertMany(inscriptions: Inscription[], chunkSize = getChunkSize()) {
```

Comment:
`ignoreDupplicateErrors` is still used on line 50. Shouldn't we switch to inline too?

Reply:
my bad, lets keep using the function.